### PR TITLE
Add uk.co.mrbenshef.TeX-Match

### PR DESCRIPTION
**Pull request by mrbenshef**
https://github.com/mrbenshef/TeX-Match

### CODE REVIEW

**Comment by mrbenshef** [comment]
bot, build uk.co.mrbenshef.TeX-Match

File: uk.co.mrbenshef.TeX-Match.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
+    "app-id": "uk.co.mrbenshef.TeX-Match",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
```
Comment (LeandroStanger) [inline uk.co.mrbenshef.TeX-Match.json:4]: Update runtime to 20.08
```suggestion
    "runtime-version": "20.08",
```

**Comment by mrbenshef** [comment]
bot, build uk.co.mrbenshef.TeX-Match

**Comment by barthalion** [comment]
bot, build uk.co.mrbenshef.TeX-Match

**Pull request closed without merging**